<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.walletapplication.payme.service.AccountOpsTest" tests="10" skipped="0" failures="0" errors="0" timestamp="2024-01-14T16:09:21" hostname="nishi" time="0.381">
  <properties/>
  <testcase name="testDeleteAccount2()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.174"/>
  <testcase name="testDeleteAccount3()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.031"/>
  <testcase name="testGetAccount()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.013"/>
  <testcase name="testUpdateAccount()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.013"/>
  <testcase name="testGetAccount2()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.008"/>
  <testcase name="testGetAccount3()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.007"/>
  <testcase name="testDeleteAccount()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.009"/>
  <testcase name="testUpdateAccount2()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.008"/>
  <testcase name="testUpdateAccount3()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.008"/>
  <testcase name="testUpdateAccount4()" classname="com.walletapplication.payme.service.AccountOpsTest" time="0.069"/>
  <system-out><![CDATA[21:39:21.216 [Test worker] ERROR AccountOps -- Wallet does not exist: 42
21:39:21.246 [Test worker] INFO AccountOps -- Wallet reterived successfully, wallet id 42
21:39:21.267 [Test worker] INFO AccountOps -- Wallet updated successfully, wallet id 42
21:39:21.281 [Test worker] ERROR AccountOps -- Wallet does not exist: 42
21:39:21.306 [Test worker] INFO AccountOps -- Wallet deleted successfully, wallet id 42
21:39:21.330 [Test worker] ERROR AccountOps -- Wallet does not exist: 42
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
